### PR TITLE
fix: dashboards

### PR DESCRIPTION
**Pull request by staticvoid255**
Fix datasource refs in dashboards, restore pod and deployment dashboards for admins, restore istio dashboards for teams.
This will allow drill-down to work, fixing redkubes/unassigned-issues#317

## Checklist

- [ ] Architecture Design Records have been added as `adr/*.md` and appended to list in `adr/_index.md`, if applicable.
- [ ] The `values-schema.yaml` file and `test/**` fixtures have been updated to reflect code changes, if applicable.
- [ ] The OpenApi Schema from redkubes/otomi-api project is compatible with definitions from `values-schema.yaml` file, if applicable.
- [ ] Helm releases are meeting otomi's baseline security policies, if applicable.
- [ ] Helm chart and helmfile changes are tested against upgrade scenario, if applicable.

### CODE REVIEW

**Comment by staticvoid255** [comment]
Sander pointed out another issue with the Velero dashboards, so I have converted back to draft

File: charts/grafana-dashboards/gatekeeper-teams/opa-gatekeeper-teams.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     {
       "datasource": {
         "type": "prometheus",
-        "uid": "P686CB814F81609FC"
```
Comment (j-zimnowoda) [inline charts/grafana-dashboards/gatekeeper-teams/opa-gatekeeper-teams.json:61]: successfully tested

File: charts/grafana-dashboards/istio-teams/service-dashboard.json [hunk 2]
```diff
@@ -3,7 +3,10 @@
     "list": [
       {
         "builtIn": 1,
-        "datasource": "-- Grafana --",
```
Comment (j-zimnowoda) [inline charts/grafana-dashboards/istio-teams/service-dashboard.json:6]: I see all the services BUT I should see only the one belonging to a team
image

Comment (j-zimnowoda) [reply charts/grafana-dashboards/istio-teams/service-dashboard.json:6]: image

File: charts/grafana-dashboards/trivy-teams/trivy-teams.json [hunk 3]
```diff
@@ -24,7 +24,6 @@
   "description": "Dashboard to view trivy published metrics\r\nhttps://aquasecurity.github.io/trivy-operator/v0.1.5/integrations/metrics/",
   "editable": true,
   "fiscalYearStartMonth": 0,
-  "gnetId": 16652,
```
Comment (j-zimnowoda) [inline charts/grafana-dashboards/trivy-teams/trivy-teams.json:27]: I can't see this dashboard in a team
image

P.S.
Tracy has been enabled

File: charts/grafana-dashboards/k8s-admin/compute-cluster.json [hunk 4]
```diff
@@ -650,7 +650,7 @@
             "link": true,
             "linkTargetBlank": false,
             "linkTooltip": "Drill down to pods",
-            "linkUrl": "/d/85a562078cdf77779eaa1add43ccec1e/k8s-resources-namespace?var-datasource=$datasource&var-cluster=$cluster&var-namespace=$__cell_1",
+            "linkUrl": "/d/a87fb0d919ec0ea5f6543124e16c42a5/k8s-resources-namespace?var-datasource=$datasource&var-cluster=$cluster&var-namespace=$__cell_1",
```
Comment (j-zimnowoda) [inline charts/grafana-dashboards/k8s-admin/compute-cluster.json:653]: successfully tested

File: charts/grafana-dashboards/k8s-admin/compute-deployment.json [hunk 5]
```diff
@@ -0,0 +1,829 @@
+{
```
Comment (j-zimnowoda) [inline charts/grafana-dashboards/k8s-admin/compute-deployment.json:1]: successfully tested 
image

File: charts/grafana-dashboards/k8s-admin/compute-ns.json [hunk 6]
```diff
@@ -363,7 +363,7 @@
             "link": true,
             "linkTargetBlank": false,
             "linkTooltip": "Drill down",
-            "linkUrl": "/d/a164a7f0339f99e89cea5cb47e9be617/k8s-resources-workload?var-datasource=$datasource&var-cluster=$cluster&var-namespace=$namespace&var-workload=$__cell&var-type=$__cell_2",
+            "linkUrl": "/d/a87fb0d919ec0ea5f6543124e16c42a5/kubernetes-compute-resources-namespace-workloads?var-datasource=$datasource&var-cluster=$cluster&var-namespace=$namespace&var-workload=$__cell&var-type=$__cell_2",
```
Comment (j-zimnowoda) [inline charts/grafana-dashboards/k8s-admin/compute-ns.json:366]: successfully tested 

image

File: charts/grafana-dashboards/k8s-admin/compute-pods.json [hunk 7]
```diff
@@ -0,0 +1,574 @@
+{
```
Comment (j-zimnowoda) [inline charts/grafana-dashboards/k8s-admin/compute-pods.json:1]: successfully tested 
image

File: charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/k8s-resources-cluster.yaml [hunk 8]
```diff
@@ -20,5 +20,5 @@ metadata:
...
...epeatIteration":null,"repeatRowId":null,"showTitle":true,"title":"Storage IO - Distribution","titleSize":"h6"}],"schemaVersion":14,"style":"dark","tags":["kubernetes-mixin"],"templating":{"list":[{"current":{"text":"default","value":"default"},"hide":0,"label":"Data Source","name":"datasource","options":[],"query":"prometheus","refresh":1,"regex":"","type":"datasource"},{"allValue":null,"current":{"text":"","value":""},"datasource":"$datasource","hide":`}}{{ if .Values.grafana.sidecar.dashboards.multicluster.global.enabled }}0{{ else }}2{{ end }}{{`,"includeAll":false,"label":null,"multi":false,"name":"cluster","options":[],"query":"label_values(up{job=\"kubelet\", metrics_path=\"/metrics/cadvisor\"}, cluster)","refresh":2,"regex":"","sort":1,"tagValuesQuery":"","tags":[],"tagsQuery":"","type":"query","useTags":false}]},"time":{"from":"now-1h","to":"now"},"timepicker":{"refresh_intervals":["5s","10s","30s","1m","5m","15m","30m","1h","2h","1d"],"time_options":["5m","15m","1h","6h","12h","24h","2d","7d","30d"]},"timezone": "`}}{{ .Values.grafana.defaultDashboardsTimezone }}{{`","title":"Kubernetes / Compute Resources / Cluster","uid":"efa86fd1d0c121a26444b636a3f509a8","version":0}`}}
```
Comment (j-zimnowoda) [inline charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/k8s-resources-cluster.yaml:23]: successfully tested 
image

File: charts/grafana-dashboards/gatekeeper-teams/opa-gatekeeper-teams.json (hunk 1 above)
Comment (j-zimnowoda) [reply charts/grafana-dashboards/gatekeeper-teams/opa-gatekeeper-teams.json:61]: image

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
Thanks for fixing the integration of Grafana Dashboards.
I checked all of them and provided some screenshots.
For this PR there are two things that need to changed:
1. Istio-team dashboards are not very helpful for a regular user and they do not limit the scope therefor I propose to not add them to team grafana
2. I cant's see Trivy dashboard, could you double check?

**Review by j-zimnowoda — APPROVED** [review]

File: charts/grafana-dashboards/istio-teams/service-dashboard.json (hunk 2 above)
Comment (j-zimnowoda) [reply charts/grafana-dashboards/istio-teams/service-dashboard.json:6]: istio dashboard has been removed for teams

**Pull request merged**